src/node_options.cc

```diff
@@ -320,10 +320,6 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::userland_loader,
             kAllowedInEnvironment);
   AddAlias("--loader", "--experimental-loader");
-  AddOption("--experimental-modules",
-            "",
-            &EnvironmentOptions::experimental_modules,
-            kAllowedInEnvironment);
```

Comment:
If this is a no-op, then you can remove the `EnvironmentOptions::experimental_modules` field and make this use NoOp like we do for `experimental-abortcontroller`. But yeah, just like in #38968, we don't get much value of removing this flag entirely.